### ISSUE TITLE
Write an IntelliJ plugin for cargo-inspect

### ISSUE THREAD

**Issue by mre** [mre/cargo-inspect#15]
Similar to [cargo-inspect-vscode](https://github.com/mre/cargo-inspect-vscode), it would be nice to have a plugin to run `cargo-inspect` from IntelliJ.
I never wrote an IntelliJ plugin, but if somebody wants to tackle that, [here](https://www.jetbrains.org/intellij/sdk/docs/basics/getting_started.html) would be the starting point in the official plugin documentation.

**Comment by linuxtim** [comment]
I thought that it might be useful to implement this in an IDE-agnostic way using the Language Server Protocol, so I've opened an issue there to see if that's feasible: https://github.com/Microsoft/language-server-protocol/issues/689

**Comment by tim-seoss** [comment]
Blocked on this feature request (which seems to have backing, but still open as of now): https://github.com/Microsoft/language-server-protocol/issues/336

**Comment by mre** [comment]
Thanks for the update. 👍
